### PR TITLE
add missing whitespace around arithmetic operator

### PR DESCRIPTION
**Pull request by MarkusHackspacher**
http://stackoverflow.com/questions/18171560/does-pep-require-whitespace-around-operators-in-function-arguments

### CODE REVIEW

File: horizons/util/python/decorators.py [hunk 1]
```diff
@@ -97,12 +97,11 @@ def __call__(self, *args, **kwargs):
 					del self.cache_dates[key]
 					return self(*args, **kwargs)
 			else:
-				self.cache_dates[key] = time.time() # new entry
+				self.cache_dates[key] = time.time()  # new entry
```
Comment (squiddy) [inline horizons/util/python/decorators.py]: We only have a handful of lines with inline comments that are prefixed with two spaces. So this and the other change below need to go.

Comment (MarkusHackspacher) [inline horizons/util/python/decorators.py]: In the file are already inline comments with more then one space:
https://github.com/unknown-horizons/unknown-horizons/blob/master/horizons/util/python/decorators.py#L117
https://github.com/unknown-horizons/unknown-horizons/blob/master/horizons/util/python/decorators.py#L136

Comment (squiddy) [inline horizons/util/python/decorators.py]: Even more so, using more than two spaces as well. So what's the reasoning behind using two spaces then?

Before I wrote the comment, I did a simple check across the codebase. We have far more inline comments with one space than two. So in my opinion, that's the way to go.

Comment (MarkusHackspacher) [inline horizons/util/python/decorators.py]: Python's official style guide, [PEP 8](http://www.python.org/dev/peps/pep-0008/), is very clear about this issue:
```
Inline comments should be separated by at least two spaces from the statement.
They should start with a # and a single space.
```
UH use PEP 8, but not everything implemented, see point
 -  **String formatting** should prefer the Python3-style format() over the % operator

Comment (squiddy) [inline horizons/util/python/decorators.py]: Actually, we recommend PEP-8 as style guide, but we don't enforce it (that's why I opened #2482). At least not according to the Wiki. And I'm pretty sure we've had the discussion about inline comments before.

While we're at it, allow me me to quote PEP-8 too:

> A style guide is about consistency. Consistency with this style guide is important. Consistency within a project is more important. Consistency within one module or function is the most important.

https://www.python.org/dev/peps/pep-0008/#a-foolish-consistency-is-the-hobgoblin-of-little-minds

You're arguing for **Consistency with this style guide is important**, whereas I actually looked at the rest of the codebase to see that (apparantly) most inline comments use one space. You keep ignoring that statement.

Comment (squiddy) [inline horizons/util/python/decorators.py]: Previous discussion about inline comments: https://github.com/unknown-horizons/unknown-horizons/pull/2290

Comment (MarkusHackspacher) [inline horizons/util/python/decorators.py]: Thank you for find the old statement from @ChrisOelmueller. I think I make it better for code reading with a good example, but this end with

> I don't see the point in arguing about this longer.

In my opinion, we should talk about it a bit. Even if it is annoying for you. *joke
I not ignoring that statement, in my first answer was the are two other line the agrees with the pep8,  in my first was make the file consistent.

> rest of the codebase to see that (apparantly) most inline comments use one space

I think logical: consistent = all the same = true,
most all the same = not consistent = false

Comment (squiddy) [inline horizons/util/python/decorators.py]: I'm not going to comment here further. I requested changes. If you apply them, I'm happy to merge this.

If you really feel this needs more discussion, even though 2! years ago we already had that discussion and made our (two long term maintainers) point clear, open an issue. But I won't tolerate you sneaking your style changes into unrelated pull-requests.

**Comment by MarkusHackspacher** [comment]
I remove second space before inline comment.

**Comment by squiddy** [comment]
Thank you.

**Pull request merged**